lib/gollum/public/gollum/javascript/editor/gollum.editor.js.erb

```diff
@@ -405,43 +398,6 @@
       return null;
     },
 
-
-    /**
-     *  loadFor
```

Comment:
Are we sure that this is deprecated and no longer used?

Reply:
Define "sure" ;)

Reply:
OK.